Pod/Classes/VOKCoreDataManager.m

```diff
@@ -6,6 +6,10 @@
 #import "VOKCoreDataManager.h"
 #import "VOKCoreDataManagerInternalMacros.h"
 
+#import <ILGClasses.h>
```

Comment:
I `pod install`ed locally and didn't commit/push those changes, since I wanted the diff to be the relevant bits and since I'd usually try to separate out the `pod install` into its own PR (but that won't really be sensible here).